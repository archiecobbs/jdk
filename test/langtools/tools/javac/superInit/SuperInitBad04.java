/*
 * @test /nodynamiccopyright/
 * @bug 8193760
 * @summary Test invalid reference to "this" prior to superclass construction
 *
 * @compile/fail/ref=SuperInitBad04.out -XDrawDiagnostics SuperInitBad04.java
 */
class SuperInitBad04 {

    SuperInitBad04() {
        super.hashCode();
        super();
    }
}
