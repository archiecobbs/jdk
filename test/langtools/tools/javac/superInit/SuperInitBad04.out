SuperInitBad04.java:11:9: compiler.err.cant.ref.before.ctor.called: super
1 error
